types/cloudinary_ts_spec.ts

```diff
@@ -1148,6 +1148,15 @@ cloudinary.v2.provisioning.account.user_group_users(
 
     });
 
+// $ExpectType Promise<AnalyzeResponse>
+cloudinary.v2.analysis.analyze('https://example.com', 'custom', {
+    analyze_parameters: {
+        custom: {
+            model_name: 'my_model',
+            model_version: 1
+        }
+    }
+});
```

Comment:
What does this do?

Reply:
If you take a look at the comment one line above - that's the assertion. It verifies if `analyze_uri` returns a Promise with `AnalyzeResponse`